packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.spec.tsx

```diff
@@ -213,4 +221,29 @@ describe('<SampleTypePropertiesPanel/>', () => {
 
         expect(tree).toMatchSnapshot();
     });
+
+    test('allowTimepointProperties', async () => {
```

Comment:
this test case name is a bit misleading since I believe. would it make more sense to have it be related to "auto-link data to study" in some way? They checks looks to be based on both the `allowTimepointProperties` prop of the domainDesign and then also based on the `appPropertiesOnly` prop.

Reply:
Good point! Updated name and also added some explanatory comments